### PR TITLE
Styling the Home page Feature

### PR DESCRIPTION
**Pull request by Haumer**
- changed the h1 to h2 and reformatted the text with lorem ipsum

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,12 +1,6 @@
...
-  <h1>I AM IN!!!!!!!!!!!!!!!</h1>
-  your email is: <%= current_user.email %>
-<% else %>
-  <h2>I AM NOT !!!!!!!!!!!</h2>
-<% end %>
-
-
+<div class="container">
+  <div class="col-sm-10">
+    <h2>Welcome to this awesome app!</h2>
```
Comment (Haumer) [inline app/views/pages/home.html.erb]: hmm not sure about the text!

Comment (KierynMo) [reply app/views/pages/home.html.erb]: looks good

Comment (KierynMo) [reply app/views/pages/home.html.erb]: please change h2 to h3

**Comment by KierynMo** [comment]
did the text get bigger?

**Comment by Haumer** [comment]
yes mate!

**Pull request merged**